doc/source/dev/reviewer_guidelines.rst

```diff
@@ -116,4 +115,67 @@ the PR page.
 Assuming you have your :ref:`development environment<development-environment>`
 set up, you can now build the code and test it.
 
+.. _saved-replies:
+
+Standard replies for reviewing
+==============================
+
+It may be helpful to store some of these in GitHub's `saved
+replies <https://github.com/settings/replies/>`_ for reviewing:
+
+**Usage question**
+    ::
```

Comment:
```suggestion
    
.. code-block:: md
```

By default, `::` tries to highlight the syntax using the default `highlight_language`, which is `python3`. By replacing these with explicit `code-blocks` you would get two advantages:

 1) It would be clear that these sections were indeed intended to be markdown, and
 2) the syntax highlighting would be correct in the rendered html

Reply:
It looks better now.. :)